v63003/u_inref/gtm8735.csh

```diff
@@ -66,13 +66,22 @@ $DSE dump -file|&$grep "Read Only"
 
 echo '# Displaying status of gtmhelp databases to verify files have read-only permissions for user/group/other'
 $gtm_tst/com/lsminusl.csh $ydb_dist/*.dat|$tst_awk '{print $1,$9}'
-
 foreach dir ($ydb_dist/*.gld)
 	echo "# Displaying status of "$dir
 	setenv ydb_gbldir $dir
 	$DSE dump -file |& $grep "Read Only"
 	echo "# Attempting to change to no read only"
-	$MUPIP SET -region DEFAULT -NOREAD_ONLY
+	$MUPIP SET -region DEFAULT -NOREAD_ONLY >>& temp.out
+	# Occasionally there is a READONLYLKFAIL Error instead of the desired DBFILEOPERR. We have the program retry
+	# if it encounters this
+	set x = `cat temp.out |& $grep READONLYLKFAIL`
```

Comment:
Please avoid duplication in line 77 and 81. You can use a "while (1)" (i.e. infinite loop) construct.

Also, the grep can be simplified by doing "$grep READONLYLKFAIL temp.out > /dev/null" and checking "if ($status)" in the next line. $status will be 0 if a match occurs and non-zero otherwise.